graylog2-web-interface/src/components/common/Icon/Icon.tsx

```diff
@@ -29,6 +29,11 @@ const sizeMap = {
   '3x': '3.45em',
   '4x': '4.60em',
   '5x': '5.75em',
+  '6x': '6.90em',
```

Comment:
Do we really need all of these? If we just need the `10x` size, we might not want to add `6x` - `9x` to reduce the amount of different icon sizes that can be used.

Reply:
Now we don't need all of them. I just added a round number of sizes to reach x9, which I need. I can replace it with huge if you think it's too much

Reply:
Yes, I think we should keep only which are actually in use.